packages/manager/apps/pci-databases-analytics/src/pages/services/create/_components/OrderFunnel.component.tsx

```diff
@@ -419,14 +431,28 @@ const OrderFunnel = ({
                 />
               )}
             </CardContent>
-            <CardFooter className="flex justify-between">
+            <CardFooter className="flex flex-col gap-2 justify-between">
               <Button
                 data-testid="order-submit-button"
                 className="w-full"
                 disabled={isPendingAddService || isProjectDiscoveryMode}
               >
                 {t('orderButton')}
               </Button>
+              <Button
+                type="button"
+                className="w-full break-words whitespace-normal"
+                mode="ghost"
+                onClick={handleApiTerraformDialog}
```

Comment:
I'm not sure you need `handleApiTerraformDialog` anymore as you're only setting the value of your state
```suggestion
                onClick={()=> setApiTerraformDialogOpen(true)}
```